recipes-browser/wpewebkit/wpewebkit_2.40.1.bb

```diff
@@ -9,7 +9,7 @@ SRC_URI = "https://wpewebkit.org/releases/${BPN}-${PV}.tar.xz;name=tarball \
 
 SRC_URI[tarball.sha256sum] = "c6b25e168b70f2121305ed078d0790e0aa4b0c73fce44e32ed42d4e5dd137ccb"
 
-DEPENDS += " libwpe"
+DEPENDS += " libwpe unifdef-native libavif"
```

Comment:
Thanks for your submission, Wouter.

unifdev-native is already added later, at the end of the file, around line 38.

Reply:
Ah, missed that one. I updated the PR